opentreemap/treemap/js/src/lib/geocoderUi.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
```

Comment:
@jwalgran, I fixed (1) and (3) but could not reproduce (2). Ready for another look.